setup.cfg

```diff
@@ -40,3 +40,6 @@ ignore =
     D401,
 
 exclude = api_pb2.py
+
+[codespell]
+check-filenames =
```

Comment:
Is this intended to be empty?

Reply:
Yes - that is the (admittedly weird) way to specify a Boolean flag.